app/models/loc_id.rb

```diff
@@ -0,0 +1,5 @@
+class LocId < ActiveRecord::Base
+  belongs_to :assorted_object, polymorphic: true
+  attr_accessible :assorted_object, :assorted_object_id, :assorted_object_type
```

Comment:
I suggest to rename this to `attr_accessible :object, :object_id, :object_type `